### ISSUE TITLE
FR: Include arm32v7 in releases

### ISSUE THREAD

**Issue by emilm** [microsoft/ApplicationInsights-Java#1490]
Hello!

I wonder if it's possible to include arm32v7 in the releases so they can work on for example PI 3.

Thanks!

**Comment by trask** [comment]
hi @emilm! have you tried the 3.0 javaagent on PI 3? does it give errors?

**Comment by emilm** [comment]
Hello @trask! Thanks for chiming in, I guess I need some extra components:
```Error occurred during initialization of VM
Could not find agent library instrument on the library path, with error: libinstrument.so: cannot open shared object file: No such file or directory
Module java.instrument may be missing from runtime image. ```

**Comment by emilm** [comment]
@trask  I am sorry. It was the missing java.instrument. The first error I got must have been some corruption of the docker container misleading me to think it was dependent on arm32v7 specific native .so files within the jar.

But I have another issue. I run this in a docker container and I wonder what app insights needs of rights, like fs mappings to /dev, or /proc or whatnot:

```
java.net.UnknownHostException: <redacted>: <redacted>: Name or service not known
        at java.base/java.net.InetAddress.getLocalHost(Unknown Source)
        at com.microsoft.applicationinsights.common.CommonUtils.getHostName(CommonUtils.java:46)
        at com.microsoft.applicationinsights.TelemetryClient.createInitializedContext(TelemetryClient.java:511)
        at com.microsoft.applicationinsights.TelemetryClient.getContext(TelemetryClient.java:98)
        at com.microsoft.applicationinsights.TelemetryClient.track(TelemetryClient.java:408)
        at com.microsoft.applicationinsights.internal.perfcounter.jvm.DeadLockDetectorPerformanceCounter.report(DeadLockDetectorPerformanceCounter.java:105)
        at com.microsoft.applicationinsights.internal.perfcounter.PerformanceCounterContainer$1.run(PerformanceCounterContainer.java:232)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
        at java.base/java.util.concurrent.FutureTask.runAndReset(Unknown Source)
        at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.net.UnknownHostException: <redacted>: Name or service not known
        at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
        at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(Unknown Source)
        at java.base/java.net.InetAddress.getAddressesFromNameService(Unknown Source)
        at java.base/java.net.InetAddress$NameServiceAddresses.get(Unknown Source)
        at java.base/java.net.InetAddress.getAllByName0(Unknown Source)
        ... 13 common frames omitted
2021-02-12 21:22:29.238Z ERROR c.m.a.i.p.FreeMemoryPerformanceCounter - Error getting FreePhysicalMemorySize
2021-02-12 21:22:29.277Z ERROR c.m.a.i.p.CpuPerformanceCounterCalculator - Error in getProcessCPUUsage
```

**Comment by trask** [comment]
hey @emilm, we just changed the way that the 3.0 agent captures these native O/S performance counters, if you have a chance, please try [3.0.3-BETA.2](https://github.com/microsoft/ApplicationInsights-Java/releases/tag/3.0.3-BETA.2)

**Comment by emilm** [comment]
Thanks, @trask ! Does this also fix the host name issue?

**Comment by trask** [comment]
hey @emilm, can you see if any of the solutions at https://stackoverflow.com/questions/1881546/inetaddress-getlocalhost-throws-unknownhostexception help with the host name issue?

**Comment by payalgaikwad42** [comment]
Hi is there any javaagent version that can work on yocto os ?

**Comment by trask** [comment]
Hi @payalgaikwad42 what error are you seeing?

**Comment by trask** [comment]
Closing, but please feel free to post further comment or create a new issue.
